.github/steps/1-create-a-branch.md

```diff
@@ -34,7 +34,7 @@ Branches allow you to separate your work from the `main` branch. In other words,
 
    ![main-branch-dropdown](/images/main-branch-dropdown.png)
 
-4. In the field, enter a name for your branch: `my-first-branch`.
+4. In the field, name your branch `my-first-branch`. In this case, the name must be `my-first-branch` to trigger the course workflow.
```

Comment:
This is great, thanks for helping add clarity to this step. 🙇🏻‍♂️

Reply:
You're welcome.
This commit is similar to 4c81193, but I cherry-picked from the skills template for consistency.